.github/workflows/ci.yaml

```diff
@@ -125,9 +125,12 @@ jobs:
         test:
           - { name: "recipes", case: "recipes" }
           - { name: "templating", case: "templating", variables: " -t escaped:\\$ba\\'d\\$gers\\ snakes" }
-          - { name: "debian (amd64)", case: "debian", variables: "-t architecture:amd64" }
-          - { name: "debian (arm64)", case: "debian", variables: "-t architecture:arm64" }
-          - { name: "debian (armhf)", case: "debian", variables: "-t architecture:armhf" }
+          - { name: "debian (bookworm amd64)", case: "debian", variables: "-t architecture:amd64 -t suite:bookworm" }
+          - { name: "debian (bookworm arm64)", case: "debian", variables: "-t architecture:arm64 -t suite:bookworm" }
+          - { name: "debian (bookworm armhf)", case: "debian", variables: "-t architecture:armhf -t suite:bookworm" }
+          - { name: "debian (trixie amd64)", case: "debian", variables: "-t architecture:amd64 -t suite:trixie" }
+          - { name: "debian (trixie arm64)", case: "debian", variables: "-t architecture:arm64 -t suite:trixie" }
+          - { name: "debian (trixie armhf)", case: "debian", variables: "-t architecture:armhf -t suite:trixie" }
```

Comment:
this massively extends the number of test jobs; Does testing trixie on arm64/armhf give us extra data? Also i'm a tad wary about it breaking CI non-deterministically

Reply:
yeah probably best to remove trixie